sv.h

```diff
@@ -199,7 +200,7 @@ SVDEF String_View sv_chop_by_delim(String_View *sv, char delim)
     return result;
 }
 
-SVDEF String_View sv_chop_by_sv(String_View *sv, String_View thicc_delim)
+SVDEF String_View sv_chop_by_sv_left(String_View *sv, String_View thicc_delim)
```

Comment:
Please add a simple alias `#define sv_chop_by_sv sv_chop_by_sv_left` in case somebody's already using the old name somewhere.

Reply:
done